tmt/steps/report/html/template.html.j2

```diff
@@ -40,53 +40,105 @@
             margin: 30px 7px 0px 7px;
         }
 
-        table {
+        #results, #filters {
             border-spacing: 7px;
         }
 
-        td, th {
+        #results td, #results th {
             padding: 0.7ex 1em;
         }
 
-        td {
+        #results td {
             background: #f8f8f8;
             border-radius: 0.5ex;
         }
 
-        td.result {
+        #results td.result {
             text-align: center;
             text-shadow: 0px 0px 5px #555;
             color: white;
         }
 
-        td.pass {
+        #results td.pass {
             background: #0a0;
         }
-
-        td.fail {
+        #results td.fail {
             background: #d30;
         }
-
-        td.info {
+        #results td.info {
             background: #58d;
         }
-
-        td.warn {
+        #results td.warn {
             background: #fc5;
         }
-
-        td.error {
+        #results td.error {
             background: #b4d;
         }
 
-        td.log {
+        #results td.log {
             word-spacing: 1ex;
         }
 
-        td.note {
+        #results td.note {
```

Comment:
Why so many style changes? Wasn't the intention to just get rid of the extra spacing and background?

```css
#filters td {
    padding: 0ex;
    background: none;
}
```

Adding just a single section targeting the `#filters` table cells should be enough. Or did I miss anything?

Reply:
I didn't actually change any of the spacing.

All I did was to limit the already-existing style to the `<table>` of results, to avoid styling the `<table>` of filter checboxes and input boxes.

Reply:
Ok, understood.